argocd/resource_argocd_application_test.go

```diff
@@ -2446,9 +2448,10 @@ resource "argocd_application" "helm_values_external" {
     }
 
     source {
+      name            = "wodpress-values"
```

Comment:
Could you also add an AttributeCheck to the test checking if the field arrived in the TF state? See [here](https://github.com/argoproj-labs/terraform-provider-argocd/blob/main/argocd/resource_argocd_application_test.go#L1080-L1083) for an example.

Reply:
added the few attributeCheck that case some cases:
- all sources name present
- only one source has name